pachyderm/templates/dash/ingress.yaml

```diff
@@ -2,7 +2,7 @@
 SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
 SPDX-License-Identifier: Apache-2.0
 */ -}}
-{{- if .Values.dash.url -}}
+{{- if and .Values.dash.enabled .Values.dash.url -}}
```

Comment:
If you wanted to be hyper-safe, you might check `.Values.dash` too, but … that is probably overkill.

Reply:
?

Reply:
To ensure that it is not falsey. But I kinda think if someone manually overrides `dash` with `null` or whatever, then … the mistake is not in the Helm chart☺

Reply:
Ah, good point! but yeah, since it's in our jsonschema / has defaults in the values.yaml I think we're ok not checking here as it _should_ always be instantiated.